merryface/pain_tracker
# Comments

Created Comments component and installed Redux Toolkit & React Redux. Comments component is hooked up with Redux store.

# To test:
Install the new dependencies by running `yarn`.

Start the project up and load up the home page at localhost:3000. Verify that there is a text area rendering on the screen. Open the Redux devtools and verify that actions are being dispatched to the store and that the state is being updated when you type into the text area.
